src/status_im/transport/handlers.cljs

```diff
@@ -22,21 +22,30 @@
   (let [{:keys [payload sig]} (js->clj js-message :keywordize-keys true)
         status-message (-> payload
                            transport.utils/to-utf8
-                           transit/deserialize
-                           (assoc :js-obj js-message))]
+                           transit/deserialize)]
     (when (and sig status-message)
       (message/receive status-message (or chat-id sig) sig cofx))))
 
-(defn- js-array->list [array]
+(defn- js-array->seq [array]
   (for [i (range (.-length array))]
     (aget array i)))
 
+(defn- add-messages-confirmation [fx js-messages]
+  ;; TODO janherich: we are ignoring `:data-store/base-tx` queue, revisit later
+  (if-let [transactions (:data-store/tx fx)]
```

Comment:
for 1-1 and pubchat messages only `:chat-received-message/add` will be added to fx, so this is not going to work

Reply:
Good point, thanks !